app/routes/admin.py

```diff
@@ -108,7 +108,8 @@ async def get_infra(
 @Authentication.admin
 async def get_refresh(
     request: Request,
-    user_jwt: Optional[str] = Cookie(None),
+    token: Optional[str] = Cookie(None),
```

Comment:
Why Optional? You're assigning defaults

Reply:
The token being optional allows the authentication system to redirect you to login instead of getting the request rejected by fastapi

Reply:
Why not just see if there's a way with FastAPI to handle the exception if the request looks wrong?

IMHO this seems like we're doing it wrong

Reply:
Fastapi handles this differently in their official examples, I don't really feel like touching the auth system today
https://github.com/HackUCF/OnboardLite/issues/123